packages/postgres-tools/src/bin/pg-describe.ts

```diff
@@ -84,17 +84,17 @@ describeDatabase(argv._[0].toString(), options).then(
 
 function printDescription(description: DatabaseDescription) {
   const formattedDescription = formatDatabaseDescription(description, { coloredOutput });
-  _.forEach(_.sortBy(_.keys(formattedDescription.tables)), (tableName) => {
+  for (const tableName of _.sortBy(Object.keys(formattedDescription.tables))) {
```

Comment:
Nit, did you consider using `Array.prototype.sort` here? I'm actually surprised the linter didn't flag this.